bindings/python/slimt.cpp

```diff
@@ -173,8 +272,6 @@ PYBIND11_MODULE(_slimt, m) {
              auto view = annotated_text.sentence(sentence_id);
              return std::string(view.data(), view.size());
            })
-      .def("word_as_range", &AnnotatedText::word_as_range)
-      .def("sentence_as_range", &AnnotatedText::sentence_as_range)
```

Comment:
We wanted these functions, the ranges are intact after UTF8 manipulations. It's the other one above that converts to a string that is broken.